api/ncf_api_flask_app/views.py

```diff
@@ -12,6 +12,7 @@
 sys.path[0:0] = [NCF_TOOLS_DIR]
 
 default_path = ""
+use_rudder_auth = True
```

Comment:
We already set Rudder as default auth method, so it easier to change it no (juste chnage the flag) 

Best way should be to have this value in a config file, but that would imply changing in the packaging too